bugwarrior/db.py

```diff
@@ -25,8 +26,9 @@
 MARKUP = "(bw)"
 
 
-DOGPILE_CACHE_PATH = os.path.expanduser(
-    os.getenv('XDG_CACHE_HOME', '~/.cache') + '/dagd.dbm')
+PYVER = '%i.%i' % sys.version_info[:2]
+DOGPILE_CACHE_PATH = os.path.expanduser(''.join([
+    os.getenv('XDG_CACHE_HOME', '~/.cache'), '/dagd-py', PYVER, '.dbm']))
```

Comment:
Yeah, just installing `pybsddb` doesn't have any effect.  `dogpile.cache` would have to invoke it explicitly and it doesn't look like it has a flag to tell it to do so.